### DISCUSSION TITLE
не работает,требует обновление

### DISCUSSION THREAD

**Discussion by parkourlife** [FarbstoffRSL/RSL-Helper#429] [Q&A]
RSL требует обновление при запуске с рейдом ,но обновлено до последней версии ,в чем может быть проблемма?

**Comment by War2022** [comment]
Дайте обновление пожалуйста!

**Comment by neorv86** [comment]
будите обновлять?

**Comment by stinger44** [comment]
Not updating via rsl updater. Latest version is v 3.3 rel 2 build 4.
What is the problem?

**Comment by ComradDart** [comment]
Farbstoff is on vacation until 16th august, he wrote about that in discord.

Разработчик в отпуске до 16 августа, он писал в дискорде об этом

**Comment by LJutii** [comment]
Ждем с нетерпением!!!
